src/services/completions.ts

```diff
@@ -127,6 +128,26 @@ namespace ts.Completions {
         return { isGlobalCompletion: isInSnippetScope, isMemberCompletion: isMemberCompletionKind(completionKind), isNewIdentifierLocation, entries };
     }
 
+    const tsKeywords: string[] = [
+        SyntaxKind.EnumKeyword,
+        SyntaxKind.AbstractKeyword,
+        SyntaxKind.AnyKeyword,
+        SyntaxKind.InferKeyword,
+        SyntaxKind.IsKeyword,
+        SyntaxKind.KeyOfKeyword,
+        SyntaxKind.ModuleKeyword,
+        SyntaxKind.NamespaceKeyword,
+        SyntaxKind.NeverKeyword,
+        SyntaxKind.ReadonlyKeyword,
+        SyntaxKind.TypeKeyword,
+        SyntaxKind.UniqueKeyword,
+        SyntaxKind.UnknownKeyword

```

Comment:
Does this mean you don't get completions for `any` and `unknown` and `never`, etc. in JSDoc?

Reply:
And that's why it needs to be context sensitive.

Reply:
Oops, forget this situation, fixed.